tool/tctl/common/tctl.go

```diff
@@ -206,6 +207,13 @@ func TryRun(commands []CLICommand, args []string) error {
 			break
 		}
 	}
+
+	if err := common.ShowClusterAlerts(ctx, client, os.Stderr, map[string]string{
+		types.AlertLicenseExpired: "yes",
+	}, nil); err != nil {
+		log.WithError(err).Warn("Failed to display cluster alerts.")
+	}
+
```

Comment:
If we add separate round trips for every new alert that we want to show outside of `on-login`, I think that'll end up being fairly messy (not to mention bad for latency).

Rather than explicitly querying for the license expired label, how about querying for all _high_ severity alerts.  That way we can have a consistent behavior across alert variants.  Show all "on-login" alerts when the user logs in.  Show all "high severity" alerts in select additional places.  Show all alerts of all severity via `alerts ls`.

Reply:
So for the case of tctl would the idea be to show all high severity on-login alerts e.g. license has expired, after every tctl command. With tctl status also showing the medium severity ones. Then an eventual addition of the alerts ls command to allow viewing all alerts?